CDP4CrossViewEditor/ViewModels/ParameterTypeSelectorViewModel.cs

```diff
@@ -46,7 +46,7 @@ public sealed class ParameterTypeSelectorViewModel : ThingSelectorViewModel
         /// <summary>
         /// Hardcoded power related parameter type short names list
         /// </summary>
-        private static readonly string[] PowerParameters = { "Redundancy", "P_on", "P_stby", "P_peak", "P_duty_cyc", "P_mean" };
+        private static readonly string[] PowerParameters = { "redundancy", "P_on", "P_stby", "P_peak", "P_duty_cyc", "P_mean" };
```

Comment:
This should be added and handled as a separate issue: I'm also having problems with the "add power parameter types" toggle:
1. after adding with the toggle, the clear ("eraser" button) does not reset toggle
1. trying to add again afterwards results in the toggle button doing nothing

Reply:
@cozminvelciu I've added a new issue: #719